README.md

```diff
@@ -31,32 +31,37 @@ implementation("dev.jamesyox:kastro:0.1.0")
 
 You can calculate the following solar events:
 
-| Event Name | Phase | Description |
-|-----:|---------------|-----------|
-|     Astronomical Twilight| Dawn, Dusk | For `AstronomicalDawn`, sun is increasing in brightness and is transitioning from `Night` toward `NauticalDawn`. For `AstronimcalDusk`, Sun is decreasing in brightness and is transitioning from `NauticalDusk` toward `Night`|
-|     Blue Hour| Dawn, Dusk | For `BlueHourDawn`, Sun is increasing in brightness and is transitioning between `NauticalDawn` toward `CivilDawn`. For `BlueHourDusk`, Sun is decreasing in brightness and is transitioning between `CivilDusk` toward `NauticalDusk`|
-|     Civil Twilight| Dawn, Dusk| For `CivilDawn`, sun is increasing in brightness and is transitioning from `NauticalDawn` toward `Day`. For `CivilDusk`, Sun is decreasing in brightness and transitioning from `Day` toward `NauticalDusk`|
-|     Day| Dawn| Sun is above the horizon|
-|     Golden Hour| Dawn, Dusk| For `GoldenHourDawn`, Sun is increasing in brightness and is transitioning between `CivilDawn` toward `Day`. For `GoldeHourDusk`, Sun is decreasing in brightness and is transitioning between `Day` toward `CivilDusk`|
-|     Nadir| Dusk| Opposite of Noon. Sun is at its lowest point below the horizon|
-|     Nautical Twilight| Dawn, Dusk| For `NauticalDawn`, sun is increasing in brightness and is transitioning from `Night` toward `CivilDawn`. For `NauticalDusk`, Sun is decreasing in brightness and is transitioning from `CivilDusk` toward `Astronomical`|
-|     Night | Dusk | Sun is below the horizon|
-|     Noon| Dawn| Opposite of Nadir. Sun is at its highest point above the horizon|
-|     Sunrise| Dawn| Sun's top edge first rises above the horizon|
-|     Sunset| Dusk| Sun's top edge completely disappears below the horizon|
+| Event | Description |
+|-----:|-----------|
+|     `AstronomicalDawn`| Sun is increaing in brightness and transitioning from `Night` toward `NauticalDawn`|
```

Comment:
Since we are defining events here I think it might make sense to make the description like the following for astronomical dusk

> Astronomical Dusk has begun. 

This means that we'd also need the end events that didn't fall neatly into solar phases

`BlueHourDawnEnd` `BlueHourDuskEnd` `GoldenHourDawnEnd` `GoldenHourDuskEnd`

I am unsure if we should also describe what the twilight "phases" are though. Like we could do:

|Event|Description|
|--|--|
|AstronomicalDusk|Astronomical dusk has begun. Astronomical dusk is when the sun is decreasing in brightness and is transitioning from `NauticalDusk` towards `Night`|

Curious for your thoughts :smile: